bin/install

```diff
@@ -48,6 +48,24 @@ download_source_file() {
   local download_path=$3
   local download_url=$(get_download_url $install_type $version)
 
+  # determine if the file exists
+  http_status=$(curl -w %{http_code} -s -o /dev/null $download_url)
```

Comment:
I now see my mistake, I will update the code to parse the output and only download once.